pkg/ginkgo-reporters/polarion_reporter.go

```diff
@@ -215,8 +216,14 @@ func (reporter *PolarionReporter) SpecSuiteDidEnd(summary *types.SuiteSummary) {
 	reporter.Suite.Time = summary.RunTime.Seconds()
 	reporter.Suite.Failures = summary.NumberOfFailedSpecs
 
+	// include parallel node number in output file name in case of ginkgo parallel runs
+	outputFile := reporter.Filename
+	if reporter.ParallelNode > 0 {
+		outputFile = fmt.Sprintf(reporter.Filename, reporter.ParallelNode)
```

Comment:
I think you would have to check here if really a `%d` is included in the filename. Otherwise you will get funny file names. Just try out https://go.dev/play/p/JJeEVzRAPAE.

Why not just appending the node number like we do in kubevirt/kubevirt?

Reply:
Something like this?

```suggestion
		outputFile = filePath.Join(reporter.ParallelNode, reporter.Filename)
```

Reply:
Or do you mean appending like this:
```suggestion
		outputFile = strings.Join(reporter.Filename, reporter.ParallelNode)
```